packages/playback-core/src/index.ts

```diff
@@ -166,10 +166,24 @@ export const getStreamInfoFromSrcAndType = async (src: string, type?: MediaTypes
 export const updateStreamInfoFromSrc = async (
   src: string,
   mediaEl: HTMLMediaElement,
-  type: MediaTypes | '' = getType({ src })
+  type: MediaTypes | '' = getType({ src }),
+  customElement: HTMLMediaElement
 ) => {
   const { streamType, targetLiveWindow, liveEdgeStartOffset } = await getStreamInfoFromSrcAndType(src, type);
 
+  if (streamType === StreamTypes.ON_DEMAND && src.endsWith('.mp4') && !src.includes(MUX_VIDEO_DOMAIN)) {
```

Comment:
it has to be a a Mux stream url right?
use the existing `toPlaybackIdFromSrc()` function